tools/zbkey

```diff
@@ -111,7 +111,7 @@ DATA_KEY_SEQNUM = '\x03'
 # Command Line Processing
 while len(sys.argv) > 1:
     op = sys.argv.pop(1)
-    if op == '-f':
+    if op == '-f' or op == '-c':
```

Comment:
zbkey should be moved to argparse, but we don't need to address that in this PR.

Reply:
I don't understand what you're trying to tell me.
I changed the -f parameter to -c to keep the parameters of the tools the same (in any other tool the channel is specified with -c, but not here). I kept the -f parameter for backwards compatibility.

Reply:
I'm just flagging that in some other update (doesn't have to be done here), we should be using argparse instead of sys.argv.pop. This code must have missed the prior update that the other tools got.